src/lib/java/de/teamlapen/lib/lib/util/SimpleSpawnerLogic.java

```diff
@@ -153,11 +168,13 @@ public void updateSpawner() {
 
                     if (UtilLib.spawnEntityInWorld(getSpawnerWorld(), getSpawningBox(), entity, 1)) {
                         onSpawned(entity);
-                        flag = true;
+                        flag1 = true;
                     }
                 }
-                if (flag) {
+                if (flag1) {
                     this.resetTimer();
+                    this.spawnedToday++;
+                    this.spawnedLast = getSpawnerWorld().getWorldTime();
```

Comment:
Not sure how worldTime behaves (it might become larger than 24000). Furthermore worlTime does not change if the day night cycle is disabled.
So just to be sure I would use getTotalWorldTime % 24000 here (as well as above).
Should probably be worldTime % 24000. Otherwise the